### ISSUE TITLE
App does not open

### ISSUE THREAD

**Issue by ElPichiii** [Rem0o/FanControl.Releases#2697]
I was on v191 of the app, i updated to v199 and now the app doesnt open, im on w11 24h2

**Comment by Rem0o** [comment]
Specs?

**Comment by ElPichiii** [comment]
Intel Core i5-9600FK
Radeon RX 580
16 GB RAM DDR4

**Comment by Rem0o** [comment]
Any logs.txt or a crash event in eventviewer?

**Comment by ElPichiii** [comment]
[crash.zip](https://github.com/user-attachments/files/16851729/crash.zip)
idk if you mean this

**Comment by Rem0o** [comment]
Yes for eventviewer, but next to the executable, there should be a log.txt file.

**Comment by ElPichiii** [comment]
[log.txt](https://github.com/user-attachments/files/16851908/log.txt)
tried my best, however i went back to v191 and its working well

**Comment by Rem0o** [comment]
![image](https://github.com/user-attachments/assets/09e65b01-219d-4239-be9d-e11025ff0601)

**Comment by ElPichiii** [comment]
[log.txt](https://github.com/user-attachments/files/16851942/log.txt)
im dumb bro sorry
